src/command/stanza/create.c

```diff
@@ -117,19 +113,19 @@ cmdStanzaCreate(void)
                     sourceFile = archiveInfoFileExists ? INFO_ARCHIVE_PATH_FILE_STR : INFO_ARCHIVE_PATH_FILE_COPY_STR;
                     destinationFile = !archiveInfoFileExists ? INFO_ARCHIVE_PATH_FILE_STR : INFO_ARCHIVE_PATH_FILE_COPY_STR;
 
-                    storageCopyP(
-                        storageNewReadP(storageRepoReadStanza, sourceFile),
-                        storageNewWriteP(storageRepoWriteStanza, destinationFile));
+                    // Using get and put instead of copy in case the storage is remote
+                    buffer = storageGetP(storageNewReadP(storageRepoIdx(repoIdx), sourceFile));
+                    storagePutP(storageNewWriteP(storageRepoIdxWrite(repoIdx), destinationFile), buffer);
                 }
 
                 if (!backupInfoFileExists || !backupInfoFileCopyExists)
                 {
                     sourceFile = backupInfoFileExists ? INFO_BACKUP_PATH_FILE_STR : INFO_BACKUP_PATH_FILE_COPY_STR;
                     destinationFile = !backupInfoFileExists ? INFO_BACKUP_PATH_FILE_STR : INFO_BACKUP_PATH_FILE_COPY_STR;
 
-                    storageCopyP(
-                        storageNewReadP(storageRepoReadStanza, sourceFile),
-                        storageNewWriteP(storageRepoWriteStanza, destinationFile));
+                    // Using get and put instead of copy in case the storage is remote
+                    buffer = storageGetP(storageNewReadP(storageRepoIdx(repoIdx), sourceFile));
+                    storagePutP(storageNewWriteP(storageRepoIdxWrite(repoIdx), destinationFile), buffer);
```

Comment:
Same as line 117.

Reply:
fixed in https://github.com/pgbackrest/pgbackrest/pull/1326/commits/c5549dca4c0863e32760de49941b1e609eb13b2e